scripts/start.sh

```diff
@@ -92,6 +92,23 @@ else
   $PYTHON_CMD scripts/build.py
 fi
 
+# Set up a local dev instance.
+# TODO(sll): do this in a new shell.
+echo Starting GAE development server
+# To turn emailing on, add the option '--enable_sendmail=yes' and change the relevant
+# settings in feconf.py. Be careful with this -- you do not want to spam people
+# accidentally!
+
+
+($NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js start_devserver --prod_env=$FORCE_PROD_MODE --gae_devserver_path=$GOOGLE_APP_ENGINE_HOME/dev_appserver.py --clear_datastore=$CLEAR_DATASTORE_ARG --enable_console=$ENABLE_CONSOLE_ARG)&
+
+# Store the job id of the most recent background command
+# which is starting server
+JOB_ID=$!
+
```

Comment:
Minor nit: Can this variable be named something more specific? What is the name of this process spawned by the background command?